src/d3a/d3a_core/sim_results/__init__.py

```diff
@@ -1,65 +0,0 @@
-from d3a.d3a_core.util import area_name_from_area_or_iaa_name
```

Comment:
Are you really sure that this can be removed ? Was it ported to d3a_interface and forgotten to be deleted from here ?

Reply:
yes, i double checked, It was ported to the d3a-interface and not removed here.